installer/README.md

```diff
@@ -1,78 +1,59 @@
 
-## Prerequisites
+- Openshift 3.11
+- GKE (Various versions)
 
-### K8s CLI's
-You need the command line tool for your particular Kubernetes cluster:
-* [Google Cloud SDK](https://cloud.google.com/sdk/)
-* [Amazon CLI for EKS](https://eksctl.io/)
+To install the CloudFlow Installer do the following:
 
-Note: Make sure you have the latest `aws-cli` version.
+1) Clone this repo
+2) During development, the latest version of the docker image may not be pushed, so build the latest version of the container using `sbt dockerBuildAndPush`
+3) Update the `image:` label in the `test/installer-deployment.yaml` file with the new docker container path
+4) `kubectl apply -f test/installer-deployment.yaml` 
+5) Check the status of the deployments in the `cloudflow-installer` namespace
 
-### Utilities
-* [jq](https://stedolan.github.io/jq/)
-* [Helm](https://helm.sh/) *note: Cloudflow installer is currently compatible with both v2 and v3*
+Install Cloudflow (GKE)
 
-## Installation Procedure
+1) `kubectl apply -f test/cloudflowinstance_gke.yaml`
+2) Check the status of the deployments in the `cloudflow` namespace
 
-### GKE
-To install Cloudflow on GKE it is a straightforward process:
+>Note that if you are installing on `openshift` use the `cloudflowinstance_openshift.yaml` file instead.
 
-```
-# create a GKE cluster with name <cluster-name>
-$ ./create-cluster-gke.sh <cluster-name>
-# install Cloudflow in the GKE cluster
-$ ./install.sh <cluster-name> gke
-```
-Replace above `<cluster-name>` with the preferred name for your GKE cluster.
+>Note If you have issues login to the GCR container repo on Mac, please see this post on SO: https://stackoverflow.com/questions/49780218/docker-credential-gcloud-not-in-system-path
 
-### EKS
-Similarly, to install Cloudflow on EKS follow this process:
+## Relation with other projects
 
-```
-# create an EKS cluster with name <cluster-name>
-$ ./create-cluster-eks.sh <cluster-name> <aws-region>
-# install Cloudflow in the EKS cluster
-$ ./install.sh <cluster-name> eks
-```
-Replace above `<cluster-name>` with the preferred name for your EKS cluster.
+The CloudFlow Installer uses the content of a number of Helm charts.
 
-#### EFS integration with EKS
+Here the detail:
 
-Some extra considerations are needed when integrating EFS with EKS. Please make sure the user launching the cluster
-satisfies the security groups requirements explained
-[here](https://docs.aws.amazon.com/efs/latest/ug/accessing-fs-create-security-groups.html).
+[![relationship with other projects](doc-images/relationship-with-other-projects.png)](https://www.lucidchart.com/invitations/accept/0a9e1636-03d9-4b66-bb5e-3fe9a281f1e1)
 
-## Uninstall Procedure
+### Code formatting
 
-In case of a failed installation, or if you simply want to uninstall entirely, run `./uninstall-cloudflow.sh`.
+The project uses `scalafmt` for formatting the source code, if you have VS Code + Metals you do not have to do anything.
```

Comment:
This seems a little outdated (the VS code ref).

Reply:
I haven't used VS Code myself. I think @olofwalker originally added this section. It could still be valid.